src/app/client/src/app/modules/dashboard/components/list-all-reports/list-all-reports.component.ts

```diff
@@ -1,20 +1,23 @@
 import { UserService } from '@sunbird/core';
 import { IImpressionEventInput } from '@sunbird/telemetry';
 import { ResourceService, NavigationHelperService } from '@sunbird/shared';
-import { Component, OnInit, AfterViewInit } from '@angular/core';
-import { catchError, mergeMap, map, tap } from 'rxjs/operators';
+import { Component, OnInit, AfterViewInit, ViewChild, ElementRef } from '@angular/core';
+import { catchError, mergeMap, map } from 'rxjs/operators';
 import * as _ from 'lodash-es';
 import { ReportService } from '../../services';
 import { of, Observable, throwError } from 'rxjs';
 import { ActivatedRoute, Router } from '@angular/router';
 import * as moment from 'moment';
+import * as $ from 'jquery';
```

Comment:
Please remove the moment and use datejs and also try not to use jquery

Reply:
Yes I'll resolve it as a part of this sprint code changes.